frontend/src/components/basic-detail/BasicDetailParent.tsx

```diff
@@ -3,31 +3,26 @@ import {useParams} from 'react-router';
 import {connect} from 'react-redux';
 import BasicDetail from './BasicDetail';
 import {fetchTaskName} from '../redux/Actions/TaskActionName';
+import Loader from '../loader/loader';
+import './basicdetail.css';
 
 const Detail: React.FC = (props: any) => {
   const {taskId} = useParams();
-
-
   React.useEffect(() => {
     props.fetchTaskName(taskId);
     // eslint-disable-next-line
   }, []);
-
-  if (props.TaskName != null) {
+  if (props.TaskName && (props.TaskName.id).toString() === taskId) {
```

Comment:
does `?.` work here? `props.TaskName?.id.toString() === taskId` ?

Reply:
yeah it works.